src/test/java/io/getunleash/UnleashTest.java

```diff
@@ -33,12 +37,23 @@ public void setup() {
                         .appName("test")
                         .unleashAPI("http://localhost:4242/api/")
                         .environment("test")
+                        .scheduledExecutor(mock(UnleashScheduledExecutor.class))
                         .unleashContextProvider(contextProvider)
                         .build();
 
         unleash = new DefaultUnleash(config, toggleRepository, new UserWithIdStrategy());
     }
 
+    private void setMock(FeatureRepository mock) {
```

Comment:
I'm not a huge fan of this black magic. I'm guessing it is needed for setting a static field? Perhaps a package private setter with a `@VisibleForTesting` annotation would be a little clearer.